instrumentation/ruby_kafka/Appraisals

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+appraise 'ruby-kafka-1.3.0' do
+  gem 'ruby-kafka', '~> 1.3.0'
+end
+
+appraise 'ruby-kafka-1.2.0' do
+  gem 'ruby-kafka', '~> 1.2.0'
+end
+
+appraise 'ruby-kafka-1.1.0' do
+  gem 'ruby-kafka', '~> 1.1.0'
+end
+
+appraise 'ruby-kafka-1.0.0' do
```

Comment:
I don't have any strong opinions on what version we should be supporting.  Mostly just picked the first major release and worked my way up.

Reply:
🤷 which versions are supported? Generally, we prefer to only support ... supported gem versions.

Reply:
I didn't see a formal statement about their supported versions, and I've added support to cover all the minor revisions of the latest major version (1).  I think this is sufficient for now.